src/ansys/fluent/core/session.py

```diff
@@ -328,7 +332,7 @@ def exit(self, **kwargs) -> None:
         self._fluent_connection.exit(**kwargs)
 
     def force_exit(self) -> None:
-        """Immediately terminates the Fluent session, losing unsaved progress and
+        """Immediately terminate the Fluent session, losing unsaved progress and
```

Comment:
```suggestion
        """Immediately exit the Fluent session, losing unsaved progress and
```
I'd prefer the verb be exit to match the verb in the method name.

Reply:
@PipKat there is a difference in meaning here, `exit` is a "graceful" exit, while `terminate` is an "externally forced" exit. Do you think something like the suggestion below would be better?
```suggestion
        """Forces the Fluent session to exit, losing unsaved progress and
```
Thanks for the help!

Reply:
Done. Thank you.